Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/06-NodeDisabling/01-DisableNodeAggregate_ConstraintChecks.feature

```diff
@@ -46,7 +46,7 @@ Feature: Constraint checks on node aggregate disabling
       | Key                          | Value                                  |
       | nodeAggregateId      | "i-do-not-exist"                       |
       | nodeVariantSelectionStrategy | "allVariants"                          |
-    Then the last command should have thrown an exception of type "NodeAggregateCurrentlyDoesNotExist"
+      | tag                          | "disabled"                    |
```

Comment:
I currently used the hard coded subtree tag "disabled" but we might want to force namespaced tags (i.g. `neos:disabled` or similar)

Reply:
we agreed to _not_ enforce namespaces